core/src/test/java/DifficultyTests.java

```diff
@@ -0,0 +1,47 @@
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.internal.util.reflection.Whitebox;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.GL20;
+
+import static org.junit.Assert.*;
+import java.beans.Transient;
+
+import com.mygdx.auber.Screens.ChooseDifficultyScreen;
+import com.mygdx.auber.Screens.PlayScreen;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.mygdx.auber.Auber;
+
+
+@RunWith(GdxTestRunner.class)
+public class DifficultyTests {
+    @Before
+    public void before(){
+        Gdx.gl = Mockito.mock(GL20.class);
+        //SpriteBatch = Mockito.mock
+    }
+
+    @Test
+    public void IncorrectArrestsTest() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.